models/tokens_models.go

```diff
@@ -33,7 +33,7 @@ func (t *AccessToken) GenerateSignOutRequest() *cognitoidentityprovider.GlobalSi
 		AccessToken: &t.TokenString}
 }
 
-type IdClaims struct {
+type IDClaims struct {
```

Comment:
Breaking change in exported struct?
(Plus others in this file eg. `IDToken`, `BuildSuccessfulJSONResponse` etc.